src/gui/advancedsettings.cpp

```diff
@@ -78,6 +78,8 @@ namespace
         RESOLVE_COUNTRIES,
         PROGRAM_NOTIFICATIONS,
         TORRENT_ADDED_NOTIFICATIONS,
+        NOTIFICATION_TIMEOUT_ENABLED,
```

Comment:
Why not join them into single setting? You can provide "disabled" as a "special value" of spin box and store `-1` as timeout value.

Reply:
I meant the following:
https://doc.qt.io/qt-5/qabstractspinbox.html#specialValueText-prop

Reply:
oh